src/_includes/cloud/cloud-docker-config-generator-cmds.md

```diff
@@ -0,0 +1,23 @@
+You use the following commands to generate the Docker configuration files and build your environment.
```

Comment:
Why would this excerpt be reusable? It's long and should have one place. Maybe the cheat sheet? I don't know. Seems like a poor candidate for reuse. I hear myself saying "where is that? I thought I saw it in this section..."

Reply:
It was in the ece-tools reference topic, and it needed to be in the Quick reference.  After I made it reusable, I decided to remove it from the ece-tools reference topic. So, now it doesn't need to be reusable.  However, I was also thinking that the table could be auto-generated from the code, which is why I left it in a separate table for now.